.github/workflows/release.yaml

```diff
@@ -9,64 +11,85 @@ env:
 
 jobs:
   release-binaries:
-    name: Release ${{ matrix.arch }}-${{ matrix.os }}
-    runs-on: ubuntu-20.04
+    name: Release ${{ matrix.target }}
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        arch:
-          - aarch64
-          - x86_64
-        os:
-          - unknown-linux-gnu
-          - apple-darwin
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-22.04
+          - target: aarch64-unknown-linux-gnu
+            os: ubuntu-22.04
+          - target: x86_64-apple-darwin
+            os: macos-11
+          - target: aarch64-apple-darwin
+            os: macos-11
+          - target: x86_64-pc-windows-msvc
+            os: windows-2022
     steps:
-      - uses: actions/checkout@v2
-      - uses: Swatinem/rust-cache@v1
+      - uses: actions/checkout@v3.1.0
+      - uses: actions-rs/toolchain@v1.0.6
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+          target: ${{ matrix.target }}
+      - uses: Swatinem/rust-cache@v2.2.0
         with:
           sharedKey: shared-cache
-      - name: Compile and release
-        uses: rust-build/rust-build.action@v1.2.0
+      - uses: actions-rs/cargo@v1.0.1
+        with:
+          use-cross: true
```

Comment:
```suggestion
          use-cross: false
```

Reply:
Cross is required for multiarch Linux builds. It is ignored for macos. Without using cross aarch64 will fail for linux.

Reply:
Got it, will turn it back on. Thanks for letting me know.